libraries/AP_Proximity/AP_Proximity.h

```diff
@@ -28,7 +28,9 @@
 
 #include <AP_HAL/Semaphores.h>
 
-#define PROXIMITY_MAX_INSTANCES             5   // Maximum number of proximity sensor instances available on this platform
+#ifndef PROXIMITY_MAX_INSTANCES 
```

Comment:
On second thoughts, I think we really should rename this before making it something someone can reasonably put in their hwdefs:

```suggestion
#ifndef AP_PROXIMITY_MAX_INSTANCES 
```

What do you think, @muramura ?

Reply:
I added the prefix "AP_" at the beginning.